### PR TITLE
feat: apply to hdac unit in hdac cli

### PR DESCRIPTION
**Pull request by JoowonYun**
- apply to hdac unit in hdac cli(transfer, bond, unbond, getbalnace)

### CODE REVIEW

File: x/executionlayer/client/util/util.go [hunk 1]
```diff
@@ -108,3 +93,73 @@ func GetLocalWalletInfo(valueFromFromFlag string, kb keys.Keybase, cdc *codec.Co
 
 	return infoList[0], nil
 }
+
+func UnitConverterRemovePoint(src string) (string, error) {
```
Comment (jhyeom26) [inline x/executionlayer/client/util/util.go]: How about renaming them like below?
eg:
```go
type U512 = string
type HDAC = U512
type BIGSUN = U512

func ToBigsun(from HDAC) (BIGSUN, error) {}
func ToHDAC(from BIGSUN) (HDAC, error) {}
```

Comment (yoosah) [reply x/executionlayer/client/util/util.go]: If you have some points of discussion, but it's not critical, how about apply this first and do discuss and make new pr?

Comment (JoowonYun) [reply x/executionlayer/client/util/util.go]: https://github.com/hdac-io/friday/pull/86/commits/378ce2e23b5a163d451af7014129dc8952bde533

If we are using U512, I think there will be some confusion on how to make a rest.

File: x/executionlayer/client/util/util.go [hunk 2]
```diff
@@ -108,3 +93,79 @@ func GetLocalWalletInfo(valueFromFromFlag string, kb keys.Keybase, cdc *codec.Co
 
 	return infoList[0], nil
 }
+
+type Hdac string
+type Bigsun string
+
+func ToBigsun(hdac Hdac) (Bigsun, error) {
+	src := string(hdac)
+	// validation
```
Comment (jhyeom26) [inline x/executionlayer/client/util/util.go]: How about separating validation logic and converting logic?

```go
func NewHdac(val string) Hdac {
// validation
}
func NewBigsun(val string) Bigsun {
// validation
}
```

**Pull request merged**